com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -190,17 +192,17 @@ public Color clearColor
 
 
         internal GraphicsFormat[] renderTargetFormat { get; set; }
-        RenderTargetIdentifier[] m_ColorAttachments = new RenderTargetIdentifier[] {BuiltinRenderTextureType.CameraTarget};
-        RenderTargetIdentifier m_DepthAttachment = BuiltinRenderTextureType.CameraTarget;
+        RTHandle[] m_ColorAttachments = new RTHandle[] {k_CameraTarget};
```

Comment:
Not sure how to solve this but you should be able to get RTHandle reference to camera target without allocating or maybe use `null`?